.github/workflows/test-and-publish.yaml

```
@@ -0,0 +1,97 @@
+      run: mypy src/bcra_connector
+
+  test:
+    name: Tests - Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+    - uses: actions/checkout@v3
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)